<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TouchScreen</class>
 <widget class="QWidget" name="TouchScreen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>779</width>
    <height>461</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TouchScreen</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>32</number>
   </property>
   <property name="bottomMargin">
    <number>48</number>
   </property>
   <item>
    <widget class="QLabel" name="touchscreenLabel">
     <property name="text">
      <string>TouchScreen</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="screenFrame">
     <property name="minimumSize">
      <size>
       <width>550</width>
       <height>50</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>50</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="monitorLabel">
        <property name="minimumSize">
         <size>
          <width>118</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>118</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string>monitor</string>
        </property>
        <property name="buddy">
         <cstring>monitorCombo</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="monitorCombo">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>30</height>
         </size>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="touchscreenFrame">
     <property name="minimumSize">
      <size>
       <width>550</width>
       <height>50</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>50</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="touchLabel">
        <property name="minimumSize">
         <size>
          <width>118</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>118</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string>touch id</string>
        </property>
        <property name="buddy">
         <cstring>touchscreenCombo</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="touchscreenCombo">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>30</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="deviceinfoFrame">
     <property name="minimumSize">
      <size>
       <width>550</width>
       <height>50</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>50</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>118</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>118</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string>input device</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="touchnameContent">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QPushButton" name="mapButton">
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>36</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>36</height>
        </size>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="text">
        <string>map</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="CalibrationButton">
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>36</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>36</height>
        </size>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="text">
        <string>calibration</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>268</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="leftMargin">
      <number>16</number>
     </property>
     <item>
      <widget class="QLabel" name="tipLabel">
       <property name="text">
        <string>No touch screen found</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>558</width>
       <height>150</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>monitorCombo</tabstop>
  <tabstop>touchscreenCombo</tabstop>
  <tabstop>mapButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
